source/gui/place.cpp

```diff
@@ -862,14 +862,13 @@ namespace nana
 
 				for (auto & child : children)
 				{
-					auto child_floor = child->calc_weight_floor(wd);
-
 					if (child->is_percent())
 					{
 						ratio += child->weight.real();
 					}
 					else
 					{
+						auto child_floor = child->calc_weight_floor(wd);
```

Comment:
This modification will cause malfunctions. Think about this situation.
`<><weight=60% <a><b fit>>` It doesn't adjust the widgets in `field b` to fit their contents.